UncomplicatedCustomItems/API/Utilities.cs

```diff
@@ -269,5 +341,17 @@ internal static void SummonCustomItem(ICustomItem CustomItem)
                 }
             }
         }
+
+        internal static IEnumerator<float> PainkillersCoroutine(Player player, IPainkillersData Data)
+        {
+            float TotalHealed = 0;
```

Comment:
totalHealed

Reply:
only args-given vars are camelCase, vars are PascalCase

Reply:
send me an article where this is written

Reply:
lots of people uses it

Reply:
Exiled developers not, plugin developers on nwapi and exiled not, server where i making plugins not too.

Reply:
this still is not a problem tho

Reply:
just change.